package.json

```diff
@@ -39,19 +39,26 @@
     "dev": "npm run build && vite",
     "serve": "npm run dev",
     "preview": "vite preview",
-    "test": "cd src/core && cargo test --release --features webgl2"
+    "test:webgl2": "cd src/core && cargo test --release --features webgl2",
+    "test:unit": "vitest"
   },
   "devDependencies": {
+    "happy-dom": "^8.9.0",
     "npm": "^8.19.2",
+    "typescript": "^5.0.4",
     "vite": "^4.3.8",
     "vite-plugin-css-injected-by-js": "^3.1.1",
     "vite-plugin-glsl": "^1.1.2",
     "vite-plugin-top-level-await": "^1.3.1",
     "vite-plugin-wasm": "^3.2.2",
-    "vite-plugin-wasm-pack": "^0.1.12"
+    "vite-plugin-wasm-pack": "^0.1.12",
+    "vitest": "^0.32.2"
   },
   "dependencies": {
     "autocompleter": "^6.1.3",
     "jquery": "^3.6.1"
+  },
+  "engines": {
```

Comment:
Very good question. For reference, [here is the official doc](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines).

Technically speaking, there is no need for it, since aladin-lite is developed only for the browser for now (as far as I can tell), and hence is executed in a browser environment, which is not an Node.js environment ([here is a quick article discussing the differences](https://nodejs.dev/en/learn/differences-between-nodejs-and-the-browser/)).

Specifying the `engines` in the package.json is meant to be used for Node.js environment, that is, for backends. It's similar to specifying a minimum "Python" version in `setup.py` of a Python library.

As of today, we could actually remove this line. But imagine that `aladin-lite` grows, and *some* of its features could be interesting to use in a backend server. The minimum Node.js version would be a minimum requirement to integrate aladin in such server.

Sincerely, your question made me wonder if we should keep it or not (I tend to include it as a good practice). It makes no harm to have it, but for the browsers it is useless.